AI-Social-Platform.Data/Migrations/ASPDbContextModelSnapshot.cs

```diff
@@ -298,15 +367,34 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("AspNetUserTokens", (string)null);
                 });
 
-            modelBuilder.Entity("AI_Social_Platform.Data.Models.Media", b =>
```

Comment:
@BagriyanBorisov  is this delete change expected?

Reply:
It wasn't expected but i believe now everything is good after the last commit

Reply:
i think the reason was because i have not applied the latest migrations on my local db